### PR TITLE
EIFF: Include GTFS identifier

### LINKED ISSUE #669: FeedEater: Lookup entities by identifier
Match Stops / Routes / RSPs to Transitland entities by GTFS identifiers.

Necessary for preserving changes in created StopPlatforms.

### PR DESCRIPTION
**Pull request by irees**
WIP

Resolves #669

### CODE REVIEW

File: spec/models/concerns/is_an_entity_imported_from_feeds_spec.rb [hunk 1]
```diff
@@ -22,6 +22,53 @@
...
+      payload = {
+        changes: [
+          {
+            action: "createUpdate",
+            stop: {
+              onestopId: stop.onestop_id,
+              # sha1:gtfs_id
+              # [sha1, gtfs_id]
+              # { feed_version: sha1, gtfs_id: gtfs_id }
+              # similar to includesOperators
```
Comment (irees) [inline spec/models/concerns/is_an_entity_imported_from_feeds_spec.rb]: remove notes

**Comment by irees** [comment]
replace imported_from_feed_onestop_id / imported_from_feed_version_sha1 serializer attrs with:
```
imported_from_feeds: [
  feed_onestop_id: "f-123-abc",
  feed_version_sha1: "a0b0c0",
  gtfs_id: "abc"
]
```

add scope / query parameter for gtfs_id

File: app/models/concerns/is_an_entity_imported_from_feeds.rb [hunk 2]
```diff
@@ -37,5 +38,26 @@ module IsAnEntityImportedFromFeeds
       where(id: self.all.select(:id).pluck(:id) - self.where_imported_from_active_feed_version.select(:id).pluck(:id))
     }
 
+    attr_accessor :add_feed_versions, :remove_feed_versions
```
Comment (drewda) [inline app/models/concerns/is_an_entity_imported_from_feeds.rb]: Could you make the names of these two virtual attributes a bit more descriptive? Perhaps `imported_from_feed_versions` and `not_imported_from_feed_versions?

File: app/models/json_schemas/imported_from_feed.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Transitland Datastore Stop",
```
Comment (drewda) [inline app/models/json_schemas/imported_from_feed.json]: title could use an update

File: app/models/json_schemas/imported_from_feed.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+	"type": "object",
+	"required": ["feedVersion"],
+	"additionalProperties": false,
+	"properties": {
+		"feedVersion": {
+			"type": "string"
+		},
+		"gtfsId": {
+			"type": ["string", "null"]
+		}
```
Comment (drewda) [inline app/models/json_schemas/imported_from_feed.json]: Also allow feed Onestop ID as input? Or you think it's better to just rely on the uniqueness of feed version SHA1s?

File: app/models/json_schemas/operator.json [hunk 5]
```diff
@@ -17,6 +17,22 @@
     "timezone": { "type": "string" },
     "tags": { "type": "object" },
     "geometry": { "type": "object" },
+    "addFeedVersions": {
```
Comment (drewda) [inline app/models/json_schemas/operator.json]: `importedFromFeedVersions`?

**Comment by drewda** [comment]
@irees my comments are mainly cosmetic, about parameter names and JSON schema being understanding in the future. Your call on the specifics!

File: app/models/concerns/is_an_entity_imported_from_feeds.rb (hunk 2 above)
Comment (irees) [inline app/models/concerns/is_an_entity_imported_from_feeds.rb]: imported_from_feed_versions is used an an association name, but I'll find something more descriptive.

**Pull request merged**